packages/wrangler/src/user.tsx

```diff
@@ -311,60 +328,54 @@ const TOKEN_URL = "https://dash.cloudflare.com/oauth2/token";
 const CALLBACK_URL = "http://localhost:8976/oauth/callback";
 const REVOKE_URL = "https://dash.cloudflare.com/oauth2/revoke";
 
-const LocalState: State = {};
-let initialised = false;
+let LocalState = getAuthState();
 
-// we do this because we have some async stuff
-// TODO: this should just happen in the top level
-// and we should figure out how to do top level await
-export async function initialise(): Promise<void> {
+/**
+ * Compute the current auth state.
+ */
+function getAuthState(): State {
```

Comment:
"state" isn't descriptive here, especially becase LocalState contains more stuff. How about `getAuthTokens()`?

Reply:
I would quite like to go a step further and split apart the OAuth state processing from the external auth token state. But I thought that would be too large a change in this PR.

The OAuth state fields that are stored in `State` really an internal implementation thing which code outside of `user.tsx` should not care about.